frontend/src/stores/redux/user/loggedUser.ts

```diff
@@ -0,0 +1,12 @@
+export const enum UserType {
```

Comment:
User type is not only for loggedUser. Create folder types and move there all types to dedicated files